### PR TITLE
Add Prism Launcher

### PR DESCRIPTION
**Pull request by Andre601**
Adds the Prism launcher to the modpack info stuff

### CODE REVIEW

File: pages/search.vue [hunk 1]
```diff
@@ -224,8 +224,9 @@
         >
         with
         <a href="https://atlauncher.com/about" target="_blank">ATLauncher</a>
-        and <a href="https://multimc.org/" target="_blank">MultiMC</a>. Pack
-        creators can reference our documentation on
+        , <a href="https://multimc.org/" target="_blank">MultiMC</a> and
+        <a href="https://prismlauncher.org" target="_blank">Prism launcher</a>.
```
Comment (wafflecoffee) [inline pages/search.vue:228]: Needs capitalization

File: pages/_type/_id.vue [hunk 2]
```diff
@@ -508,7 +508,10 @@
             <a href="https://atlauncher.com/about" target="_blank"
               >ATLauncher</a
             >
-            and <a href="https://multimc.org/" target="_blank">MultiMC</a>.
+            , <a href="https://multimc.org/" target="_blank">MultiMC</a>
+            and <a href="https://prismlauncher.org" target="_blank"
```
Comment (wafflecoffee) [inline pages/_type/_id.vue:512]: Fix Prettier please!

**Review by wafflecoffee — CHANGES_REQUESTED** [review]

File: pages/_type/_id.vue (hunk 2 above)
Comment (Andre601) [reply pages/_type/_id.vue:512]: I... don't quite get the complains it gives...

Should I change the lines 511 and 512 like this?
```html
, <a href="https://multimc.org/" target="_blank">MultiMC</a> and
<a href="https://prismlauncher.org" target="_blank"
```

**Comment by Andre601** [comment]
...and now docker fails too... cool

**Review by wafflecoffee — APPROVED** [review]
Fixed it myself

**Comment by Andre601** [comment]
> Fixed it myself

Actually didn't saw that other review comment in the UI...

**Pull request merged**